src/between.js

```diff
@@ -15,9 +15,10 @@ const _requestAnimationFrame = ( // polyfill
   || raf
 );
 
-let _prevTime = Date.now(), _time, _delta;
+let _prevTime = Date.now(), _time, _delta, _paused;
```

Comment:
Won't `_paused` stop all loops instead of just one? @hirako2000

Reply:
yes it would i thought one instance per tween... why would they share state?
I think, if that's the case, one instance per tween would  be preferable, furthermore, should move away from `Date.now()l` to some sort of `Loop /timer`instance instance, for finer grained control.